pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -3613,7 +3613,27 @@ public Pair<String, Integer> invokeControllerPeriodicTask(String tableName, Stri
 
     LOGGER.info("[TaskRequestId: {}] Periodic task execution message sent to {} controllers.", periodicTaskRequestId,
         messageCount);
-    return Pair.of(periodicTaskRequestId, messageCount);
+    return new PeriodicTaskInvocationResponse(periodicTaskRequestId, messageCount > 0);
+  }
+
+  public static class PeriodicTaskInvocationResponse {
```

Comment:
Same with this class. It is returned externally, move it so some api (or perhaps spi) package. Not sure what the convention is that we are following

Reply:
Moved to api module.